### PR TITLE
Chore[Events]: Added DEVCONF-2022 talks

### LINKED ISSUE #2421: [Event] Measuring Service Mesh Performance 101
Please add the following event to layer5.io/events.

**Event:**
- Event Title: Measuring Service Mesh Performance 101
- Event Link: https://sched.co/siKF
- Talk Description: See the event link
- Speaker: [Navendu Pottekkat](https://layer5.io/community/members/navendu-pottekkat)
---
**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs).

### PR DESCRIPTION
**Pull request by Abhijay007**
Signed-off-by: Abhijay Jain <Abhijay007j@gmail.com>

**Description**

Added DEVCONF-2022 measuring SMP and Journey to Service Meshes

This PR fixes #2421 

**Notes for Reviewers**

This PR is related to #2433 

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by leecalcote** [comment]
Only one event should be listed.

**Comment by leecalcote** [comment]
I'll let @navendu-pottekkat answer any further questions that you have @Abhijay007.

**Comment by l5io** [comment]
🚀 Preview for commit 951a042ff192f30948a9db6619882fe7bb3d5cf4 at: https://61dc9301b61e4b00ac99a9c1--layer5ng.netlify.app

File: src/collections/events/devconf-2022-measure-smp/index.mdx [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+---
+
+import { Link } from "gatsby";
+
+<p>
+  Join Layer5 at <a href="https://www.devconf.info/cz/">DevConf.cz 2022</a> as
+  our maintainers deliver talks on Measuring SMP and Introduction to Service
+  Meshes and Meshery. Bring your questions. We have answers. Engage with us in
+  our talks!
+</p>
```
Comment (navendu-pottekkat) [inline src/collections/events/devconf-2022-measure-smp/index.mdx:22]:
```suggestion
<p>
  Join Layer5 at <a href="https://www.devconf.info/cz/">DevConf.cz 2022</a> as we share on the our CNCF projects Meshery and Service Mesh Performance. Bring your questions. We have answers. Engage with us in our talks!
</p>
```

**Review by navendu-pottekkat — COMMENTED** [review]
The link should just be `/devconfcz-2022`.

The current intro line is good so let's keep it that way itself. I have made that suggestion in the review.

Rest looks good to me.

**Comment by l5io** [comment]
🚀 Preview for commit 5b18312ce51c51d23d1cf42bb39169677efa42f8 at: https://61e09df747d9a61735a5a9a6--layer5ng.netlify.app

File: src/collections/events/devconf-2022-measure-smp/index.mdx [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+upcoming: true
+# Deck: N/A
+# Video: N/A
+---
+
+import { Link } from "gatsby";
+
+<p>
+  Join Layer5 at <a href="https://www.devconf.info/cz/">DevConf.cz 2022</a> as
+  we share on the our CNCF projects Meshery and Service Mesh Performance. Bring
```
Comment (navendu-pottekkat) [inline src/collections/events/devconf-2022-measure-smp/index.mdx:19]:
```suggestion
  we share on our CNCF projects Meshery and Service Mesh Performance. Bring
```

Comment (navendu-pottekkat) [reply src/collections/events/devconf-2022-measure-smp/index.mdx:19]: Typo fix.

**Review by navendu-pottekkat — APPROVED** [review]

**Comment by l5io** [comment]
🚀 Preview for commit a0ec6971cf68eb4fb3597e9209f9f6dbb7496a9e at: https://61e18ab16da1490d1f1b2ad0--layer5ng.netlify.app

**Pull request merged**